app/models/tag.rb

```diff
@@ -0,0 +1,16 @@
+class Tag < ActiveRecord::Base
+  validates :tagname, presence: true, uniqueness: true
+
+  has_many :ex_link_tags
+  has_many :ex_links, through: :ex_link_tags
+
+  # TODO: this is repetitive from ex_links_controller list_tags!
+  # This should be method of Tag, not instance of Tag
+  def delete_unused_tags
```

Comment:
`def self.delete_unused_tags` will create a method on Tag.
Which can be called with `Tag.delete_unused_tags`